ddf-rpc-api/pom.xml

```diff
@@ -0,0 +1,294 @@
+            <version>${hamcrest-all.version}</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <version>${junit.version}</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.mockito</groupId>
+            <artifactId>mockito-all</artifactId>
+            <version>${mockito-all.version}</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>ddf.platform.security</groupId>
+            <artifactId>log-sanitizer</artifactId>
+            <version>2.23.0</version>
+        </dependency>
+        <dependency>
+            <groupId>com.connexta.jsonrpc.email</groupId>
+            <artifactId>email-service</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+
+    </dependencies>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-deploy-plugin</artifactId>
+                <version>${maven.deploy.plugin.version}</version>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-bundle-plugin</artifactId>
+                <version>3.5.1</version>
+                <extensions>true</extensions>
+                <configuration>
+                    <instructions>
+                        <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
+                        <Export-Package>
+                            com.connexta.jsonrpc
+                        </Export-Package>
+                        <Embed-Dependency>
+                            catalog-core-api-impl,
+                            platform-util,
+                            log-sanitizer
+                        </Embed-Dependency>
+                        <!--
+                        org.codice.ddf.log.sanitizer;version="[2.19,3)",
+                        -->
+                        <Import-Package>
+                            !com.google.common.util.concurrent.internal,
+                            !com.google.errorprone.annotations.*,
+                            !javax.annotation,
+                            !org.checkerframework.checker.nullness.qual,
+                            org.opengis.annotation;version="[19.1.0,21.0.0)",
+                            org.opengis.filter;version="[19.1.0,21.0.0)",
+                            org.opengis.filter.sort;version="[19.1.0,21.0.0)",
+                            org.opengis.filter.expression;version="[19.1.0,21.0.0)",
+                            org.apache.commons.lang3;version="[3.8.0,4.0.0)",
+                            org.apache.commons.lang3.tuple;version="[3.8.0,4.0.0)",
+                            ddf.security;version="[2.19,3)",
```

Comment:
See if this is removeable or if the transitive dependency powers that be require us specify this